Keycloak.Migrator.Models/JSONUser.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Keycloak.Migrator.Models
+{
+    /// <summary>
+    /// Represents a User from the JSON Migration.
+    /// </summary>
+    public class JSONUser
```

Comment:
Same as the previous comment.  Let's make the properties more consistent with C# standards and map the JSON using JsonProperty attribute.